tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -126,10 +126,18 @@ export type CardMetadataMap = Record<
   CardMetadata
 >;
 
+export enum CardSelectionState {
```

Comment:
I also realize that you are trying to name this generically enough to cover both step selection and range selection. So maybe CardSelectionEnablement. On the other hand, if you used CardStepSelectionEnablement for the name and just noted in a comment that you also use it for Range selection - that might actually be overall clearer.

Reply:
CardOverride? CardStepSelectionOverride? Or CardFeatureOverride? Not "State" if it can be avoided. Especially since it is an enum type name and not type describing a feature's overall ngrx state.

At the moment, CardFeatureOverride calls out to me the most.